cli/cmd/sync_test.go

```diff
@@ -69,6 +75,27 @@ func TestSync(t *testing.T) {
 			} else {
 				require.Contains(t, commandError.Error(), tc.err)
 			}
+
+			if tc.wantSourcePluginCache {
+				// check that source plugin was downloaded to the cache using --cq-dir
+				p := path.Join(cqDir, "plugins", "source")
+				files, err := os.ReadDir(p)
```

Comment:
Non blocking but we could use `require.FileExists` to test that the files in the directory match what we expect.
There's also `DirExists` but it doesn't check the content of a directory

Reply:
Yeah, this will be a bit more work, because we'd have to parse the config to extract the version (the directory gets named that way). Not impossible, but I think it's probably not necessary for now, we just want to check that the correct cache directory got used in general.

Reply:
Ah, yes but we could do a separate test with a dedicate config we know the names for